### PR TITLE
Add note about smart proxy workaround for ACD

### PR DESCRIPTION
**Pull request by maximiliankolb**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only)

### CODE REVIEW

File: guides/common/modules/proc_deploying-an-application-instance.adoc [hunk 1]
```diff
@@ -13,3 +13,19 @@ If the selected application instance has been deployed before, confirm that you
...
+ACD detects which {SmartProxyServer} is used to configure the application instance and its hosts.
+
+ifdef::orcharhino[]
+image::common/acd-deploying-an-application-instance-rex-target-host-orcharhino.png[Deploying an ACD Application Instance]
+endif::[]
+
+For each {SmartProxyServer}, the Ansible playbook runs to configure all hosts.
+The remote execution plug-in, which works host-based, is used to run the Ansible playbook on each {SmartProxyServer}.
+
+Therefore, the first host of the application instance is used to initiate running the Ansible playbook.
```
Comment (adamlazik1) [inline guides/common/modules/proc_deploying-an-application-instance.adoc:28]: Maybe?
```suggestion
Therefore, the first host of the application instance runs the Ansible playbook.
```

File: guides/common/modules/proc_deploying-an-application-instance.adoc [hunk 2]
```diff
@@ -13,3 +13,19 @@ If the selected application instance has been deployed before, confirm that you
...
+The *configuration jobs* page shows the first host of each {SmartProxyServer} as target host.
+The ACD component of the {SmartProxy} starts an Ansible playbook which configures all hosts connected to this {SmartProxyServer}.
```
Comment (adamlazik1) [inline guides/common/modules/proc_deploying-an-application-instance.adoc:30]:
```suggestion
The ACD component of {SmartProxy} starts an Ansible playbook which configures all hosts connected to this {SmartProxyServer}.
```

**Review by adamlazik1 — COMMENTED** [review]
Mostly looks good!

File: guides/common/modules/proc_deploying-an-application-instance.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_deploying-an-application-instance.adoc]: Applied.

File: guides/common/modules/proc_deploying-an-application-instance.adoc (hunk 2 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_deploying-an-application-instance.adoc]: Applied.

**Review by maximiliankolb — COMMENTED** [review]
Applied both suggestions and rebased to master. Ready for re-review @adamlazik1

File: guides/common/modules/proc_deploying-an-application-instance.adoc [hunk 3]
```diff
@@ -13,3 +13,19 @@ If the selected application instance has been deployed before, confirm that you
...
+ACD detects which {SmartProxyServer} is used to configure the application instance and its hosts.
+
+ifdef::orcharhino[]
+image::common/acd-deploying-an-application-instance-rex-target-host-orcharhino.png[Deploying an ACD Application Instance]
+endif::[]
+
+For each {SmartProxyServer}, the Ansible playbook runs to configure all hosts.
+The remote execution plug-in, which works host-based, is used to run the Ansible playbook on each {SmartProxyServer}.
+
+Therefore, the first host of the application instance runs the Ansible playbook.
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/modules/proc_deploying-an-application-instance.adoc]: No worries. Applied.

**Review by adamlazik1 — APPROVED** [review]
LGTM

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: :heavy_check_mark: 
   bf38786a0..8f154c7f0  3.6 -> 3.6
   82fdd3fc3..2c04da9a3  3.5 -> 3.5
   c972a4650..500b0fa9c  3.4 -> 3.4
   6e2c3e53b..7c7d11779  3.3 -> 3.3

**Pull request merged**